pkg/test/framework/components/echo/common/deployment/external.go

```diff
@@ -67,7 +68,12 @@ func (e External) build(b deployment.Builder) deployment.Builder {
 				},
 			},
 		},
-	})
+	}
+	if t.Settings().EnableDualStack {
+		config.IPFamilies = "IPv6, IPv4"
```

Comment:
Beside ["IPv6, IPv4"], shall we add ["IPv4, IPv6"] as an extra test case? or is there no business with the order of IPFamilies?

Reply:
Tough to add all combinations, but there is a plan to have a separate suite for ipv6 first dual-stack cluster , by which we can cover more scenarios. Would that be enough?

Reply:
As discussed on slack, this can be followed up in a subsequent PR